plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -363,6 +365,14 @@ func (p *Plugin) admitPVCStatus(nodeName string, a admission.Attributes) error {
 		oldPVC.Status.Conditions = nil
 		newPVC.Status.Conditions = nil
 
```

Comment:
is there any verification this particular node should have write access to the status of this particular PVC? what prevents a compromised node from writing bogus resizeStatus or allocatedResources to all PVCs? if nothing prevents that, what are the consequences if a node does that?

Reply:
We have checks https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/auth/authorizer/node/node_authorizer.go#L117 to allow `pvc.Status` update only from nodes which have valid path(relationship) to pvc.

Reply:
right. it's been too long since I've been in here :)